e2e/tests/upgrades/genesis_test.go

```diff
@@ -0,0 +1,155 @@
+package upgrades
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	cosmos "github.com/strangelove-ventures/interchaintest/v7/chain/cosmos"
+	"github.com/strangelove-ventures/interchaintest/v7/ibc"
+	test "github.com/strangelove-ventures/interchaintest/v7/testutil"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/cosmos/ibc-go/e2e/testsuite"
+	"github.com/cosmos/ibc-go/e2e/testvalues"
+)
+
+type GenesisState map[string]json.RawMessage
+
+func TestGenesisTestSuite(t *testing.T) {
+	suite.Run(t, new(GenesisTestSuite))
+}
+
+type GenesisTestSuite struct {
+	testsuite.E2ETestSuite
+	cosmos.ChainNode
```

Comment:
what's the reason for this embedding? This seems a bit odd to have a test case itself be composed of a chain node?

Reply:
Yep, it's superfluous, I'll delete it.